medineshpoudel/expense-tracker
# ET-4-DP: Implement Redux toolkit store in app.

If merged this PR will,

Setup redux toolkit in the frontend of the app. 

**Test Cases:**

1. Run the app.
2. Navigate to home page and test if the counter works.
3. Navigate to the './src/store' and see if the implementation is setup properly.
